### PR TITLE
[nim/ru] Deeper translation

### PR DESCRIPTION
**Pull request by dvska**
tested @ 
Nim Compiler Version 1.1.1 [MacOSX: amd64]

  -  Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by 1nfdev** [comment]
Ты ебанулся, дядя?

File: ru-ru/nim-ru.html.markdown [hunk 1]
```diff
@@ -169,89 +170,94 @@ var anotherArray = ["Default index", "starts at", "0"]
...
+      return Ответ.оНет
+    else: echo("Поточнее, да или нет")
+
+proc добавьСахар(колво: int = 2) =   # Значение поумолчанию 2, ничего не возвращает
+  assert(колво > 0 and колво < 9000, "Диабет ☠")
+  for a in 1..колво:
+    echo(a, " кубик...")
+
+case спрос("Сахарку?")
+of оДа:
```
Comment (1nfdev) [inline ru-ru/nim-ru.html.markdown]: I don’t know who maintained the repository, but if you don’t speak Russian, know that this is a crazy person, for example, there are sexual inclinations:

Comment (1nfdev) [reply ru-ru/nim-ru.html.markdown]: In no case do not accept this request. For many Russian-speaking people, this text will cause a microstroke.

Comment (dvska) [reply ru-ru/nim-ru.html.markdown]: Please explain what's wrong.

**Comment by ptcodes** [comment]
Hey @mxschumacher, could you delete this PR? This is a strange translation (potentially a joke). Thanks!

**Pull request closed without merging**